README.md

```diff
@@ -452,28 +452,22 @@ that can adversely effect the performance of the application.
 Same as normal Python programs, Legate programs can be run
 using Jupyter Notebook. Currently we support single node execution with
 multiple CPUs and GPUs, and plan to support multi-node execution in the future.
-We leverage Legion's Jupyter support, so you may want to refer to the  
+We leverage Legion's Jupyter support, so you may want to refer to the
 [relevant section in Legion's README](https://github.com/StanfordLegion/legion/blob/master/jupyter_notebook/README.md).
-To simplify the installation, we provide a script specifically for Legate libraries. 
+To simplify the installation, we provide a script specifically for Legate libraries.
 
 ### Installation of the Legate IPython Kernel
 
-Please install Legate, then run the following command to install the IPython
-kernel:
+Please install Legate, then run the following command to install a default
+Jupyter kernel:
 ```
-python -m legate.jupyter --json=legate_jupyter.json
+legate-jupyter
 ```
-If `--json=` is not provided, the installation script will look for a file
-named `legate_jupyter.json` in the current directory. A sample
-`legate_jupyter.json` file is provided in the legate.core source directory.
-
 If installation is successful, you will see some output like the following:
 ```
-IPython kernel: legate_kernel_nocr(Legate_SM_GPU) has been installed
+Jupyter kernel spec Legate_SM_GPU (Legate_SM_GPU) has been installed
 ```
-`Legate_SM_GPU` is the kernel name, and you will need to provide it
-when starting the Jupyter Notebook. `SM` means the kernel is only for
-shared memory execution; `GPU` means GPU support is enabled. 
+Legate_SM_GPU` is the default kernel name.
```

Comment:
Missing backtick at start of line.

Reply:
https://github.com/nv-legate/legate.core/pull/425/commits/6a24f41092bbda8de4cf2c8a203eb02040d466af